src/core/telemetry/telemetry.cpp

```
@@ -226,9 +227,9 @@ nixlTelemetry::updateMemoryDeregistered(uint64_t memory_deregistered) {
 }
 
 void
-nixlTelemetry::addXferTime(std::chrono::microseconds xfer_time, bool is_write, uint64_t bytes) {
-    const char *bytes_name = is_write ? "agent_tx_bytes" : "agent_rx_bytes";
-    const char *requests_name = is_write ? "agent_tx_requests_num" : "agent_rx_requests_num";
+nixlTelemetry::addXferTime(const std::chrono::microseconds xfer_time, const nixl_xfer_op_t operation, const uint64_t bytes) {
```

Cppcheck (2.20.0)

[style] The function 'addXferTime' is never used.

(unusedFunction)